packages/app/windows/runner/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ project(runner LANGUAGES CXX)
 # work.
 #
 # Any new source files that you add to the application should be added here.
-add_executable(${BINARY_NAME} WIN32
+add_executable(TownTalk WIN32
```

Comment:
Was this intentional?

Reply:
Loosely. I wanted to see if it would change anything. I'll probably revert it